modules/dnn/include/opencv2/dnn/version.hpp

```diff
@@ -6,7 +6,7 @@
 #define OPENCV_DNN_VERSION_HPP
 
 /// Use with major OpenCV version only.
-#define OPENCV_DNN_API_VERSION 20230620
+#define OPENCV_DNN_API_VERSION 20231225
```

Comment:
@opencv-alalek Could you tune abi-complience-checker on CI.

Reply:
> abi-complience-checker 

That result is ignored on 5.x branch